src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -49,6 +56,20 @@ public AddTaskCommand(Task task) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
+        Set<Name> persons = toAdd.getPersons();
+        //Checks if persons are in lastShownList
+        for (Name name: persons) {
+            boolean notFound = true;
+            for (Person person: lastShownList) {
+                if (person.getName().equals(name)) {
+                    notFound = false;
+                }
+            }
+            if (notFound) {
+                throw new CommandException(MESSAGE_CONTACT_NOT_FOUND);
+            }
+        }
```

Comment:
Could possibly include a check here to ensure that a person is not attached to a conflicting task.

Reply:
Okay! I will add in this constraint later. Thanks for the suggestion! 👍